tex/measure/caratheodory.tex

```diff
@@ -505,9 +507,9 @@ \section{A fourth row: Carath\'{e}odory for pre-measures}
 to $\mu$ on $\SA$, where $\SA$ is the $\sigma$-algebra generated by $\SA_0$.
 Indeed, our original goal was to get $\SB(\RR^n)$, and in fact:
 \begin{exercise}
+	\label{exer:cubes_vs_open}
```

Comment:
Apparently, the other way (without a `%` on the preceding line) introduces an unwanted space. (on my machine)

Before/after:

![image](https://github.com/vEnhance/napkin/assets/25191436/aae45558-d895-48d1-a7f3-e3d73ff7c2c2)

Reply:
Good catch.